File: lib/src/enums/asset_type.dart
/// Describes the location of the asset.
enum AssetType {
  /// Flutter asset folder (e.g. ./assets/gltf)
  /// See https://docs.flutter.dev/ui/assets/assets-and-images#specifying-assets
  flutterAsset,

  /// Documents folder for the current app
  /// This might be useful for downloading files from the internet
  documents,
}

File: ios/Classes/Enums/AssetType.swift
enum AssetType {
    case flutterAsset // Flutter asset folder (e.g. ./example/assets/gltf), path should be added to pubspec.yaml
    case documents // Documents folder for the current app
    
    static func from(index: Int) -> AssetType {
        if(index == 0){
            return .flutterAsset
        }else{
            return .documents
        }
    }
}
